docker-compose.yml

```
@@ -27,27 +38,64 @@ services:
     environment:
       DATABASE_URL: http://couchdb:5984
       DATABASE_NAME: app
-      DATABASE_USER: admin
+      DATABASE_USER: ${COUCHDB_USER:-admin}
       DATABASE_PASSWORD: ${COUCHDB_PASSWORD:?Admin password not set}
       JWT_SECRET: ${JWT_SECRET:?JWT secret not set}
```

```suggestion
      JWT_SECRET: ${JWT_SECRET}
```
in the basic profile without replication-backend, we don't need the JWT_SECRET (I think, seems to work and is not defined for these instances currently)